### DISCUSSION TITLE
Failed to install DB Browser

### DISCUSSION THREAD

**Discussion by Bee131031003** [sqlitebrowser/sqlitebrowser#3479] [Q&A]
After I downloaded "DB.Browser.for.SQLite-3.12.2-win64.msi" - [DB Browser for SQLite - Standard installer for 64-bit Windows] from 'https://sqlitebrowser.org/dl/'. When I tried to install the app, this error msg popped up. May I know how to solve this error?
<img width="278" alt="image" src="https://github.com/sqlitebrowser/sqlitebrowser/assets/88914495/14b63559-96bc-4091-b27e-ca6c1c9ec920">

**Comment by justinclift** [comment]
Oh, it sounds like your download went wrong for some reason.

I'd try it again, and/or verify the SHA256 checksum for it:

```
723d601f125b0d2402d9ea191e4b310345ec52f76b61e117bf49004a2ff9b8ae
```

A download mirror is here too, if that helps:

https://github.com/sqlitebrowser/sqlitebrowser/releases/tag/v3.12.2

**Comment by chrisjlocke** [comment]
Short answer .. we've not seen that before.  Did you run the installer as an admin user?  Right-click the .msi file and select 'run as Administrator'.
I would suspect the download is damaged .. does the file hash match the hash on the download page?
3.12.2 is quite old now, and I'd be tempted to install a nightly.  We're in the process of building a 3.12.3 release, but downloading a nightly is easier and the nightlies are fairly stable at the moment.
https://nightlies.sqlitebrowser.org/latest/

If the .msi there fails too, simply download the .zip version, extract, and run the .exe from there - technically DB4S doesn't need installing and is happy to run from the directory.

**Reply by chrisjlocke** [reply]
@Bee131031003 - any advance on this at all?

**Reply by justinclift** [reply]
Off topic, but it looks like GitHub now isn't always showing when labels are added to a discussion.

That's unfortunate, as it stops us being able to use them for properly marking issues that need to be followed up on.

**Comment by chrisjlocke** [comment]
Closing as no response.
